Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -430,13 +430,10 @@ public class CarPlayNavigationViewController: UIViewController {
             self?.carInterfaceController.popTemplate(animated: true)
             
             // TODO: Fix this Demeter violation with proper encapsulation
-            guard let uuid = self?.navigationService.eventsManager.recordFeedback() else { return }
+            guard let feedback = self?.navigationService.eventsManager.createFeedback() else { return }
```

Comment:
Now that directly managing feedback is a valid application use case, this Demeter violation becomes more relevant. Perhaps we can add a pass-through property to NavigationService or even higher up in CarPlayManager so the application doesn’t necessarily have to dig several levels down to get the events manager.

Reply:
Added a pass-through property to the `CarPlayManager`.